test/format/format_util.c

```diff
@@ -424,3 +424,14 @@ wt_wrap_close_session(WT_SESSION *session)
 
     testutil_check(session->close(session, NULL));
 }
+
+/*
+ * enable_prefetch --
+ *     Return true if prefetch should be enabled for a session.
+ */
+bool
+enable_prefetch(void)
+{
+    /* Enable prefetch 20% of the time. */
```

Comment:
What do people think of 20%?

Reply:
I think 20% is good. Two minor suggestions: Since there is a connection level prefetch setting, maybe call this function `enable_session_prefetch`. I also suggest expanding the comment just a bit to make sure the reader is clear that prefetch needs to be enabled in the connection, and then can be enabled in the session.

Reply:
Sure! Done in dcfb279d73987623e53f1d93b2b964b002f350f0